staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/config.go

```diff
@@ -37,14 +37,15 @@ type WorkEstimatorConfig struct {
 	*ListWorkEstimatorConfig     `json:"listWorkEstimatorConfig,omitempty"`
 	*MutatingWorkEstimatorConfig `json:"mutatingWorkEstimatorConfig,omitempty"`
 
-	// MinimumSeats is the minimum number of seats a request must occupy.
-	MinimumSeats uint64 `json:"minimumSeats,omitempty"`
-	// MaximumSeats is the maximum number of seats a request can occupy
+	// MinimumSeatsLimit is a lower limit on the minimum number of seats a request must occupy.
+	MinimumSeatsLimit uint64 `json:"minimumSeats,omitempty"`
```

Comment:
"minimum" already a "limit", we do not need add the word "limit" here.

Reply:
(see comment above)